Crypt.xcodeproj/project.pbxproj

```diff
@@ -364,16 +366,16 @@
 					"@executable_path/../Frameworks",
 					"@loader_path/../Frameworks",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 10.13;
-				MARKETING_VERSION = 3.3.1;
+				MACOSX_DEPLOYMENT_TARGET = 10.15;
```

Comment:
Does this prevent running on below 10.15? I don’t mind, but we need to call that out

Reply:
I'm actually not sure, I don't even know if I have machines older to test with.

Reply:
Me neither. 4.0 is a major release, I’m fine with it being 10.15+ only.

Reply:
yes, this will prevent running on < 10.15

Reply:
made it 10.14 since I'm selfish and I've still got a few.

Reply:
You monster